packages/fxa-shared/test/metrics/amplitude.js

```diff
@@ -508,6 +508,47 @@ describe('metrics/amplitude:', () => {
       assert.isTrue(result);
     });
 
+    it('success - if event_type is `fxa_pay_setup - *` and country_iso is 2 character code', () => {
```

Comment:
Nit: to keep these `it` blocks shorter/more readable, can we just say "... and country_iso is valid/invalid"?